src/adapters/mc/asb/AzureLinuxBaseline.mof

```diff
@@ -103,7 +103,7 @@ instance of OsConfigResource as $OsConfigResource5ref
 instance of OsConfigResource as $OsConfigResource6ref
 {
     ResourceID = "Ensure that the denied users for SSH are configured (CIS: L1 - Server - 5.2.18)";
-    RuleId = "1DB7D147-3A75-4637-8CDC-C119E22D0206";
+    RuleId = "1db7d147-3a75-4637-8cdc-c119e22d0206";
```

Comment:
>"1db7d147-3a75-4637-8cdc-c119e22d0206"; [](http://example.com/codeflow?start=13&length=39)

This is actually by design. The uppercase UUIDs indicate the new rules that do not have metadata in MDC.

Reply:
I see, but we will need to update the metadata for MDC for new rules.

It complicates consumers if we emit mixed lower and upper case uuids. Is there a strong reason not to unify this? We can discover rules not covered by MDC metadata again at any time by iterating and performing a lookup.

Reply:
You mean, consumers will see these UUIDs in the MDC UI? These UUIDs are not shown to consumers anywhere else as far as I know. Fine with me to correct all to lowercase, but not sure if this makes the bar for upcoming preview releases in January.